gnuradio-runtime/lib/block.cc

```diff
@@ -365,7 +365,7 @@ long block::min_output_buffer(size_t i)
 void block::set_min_output_buffer(long min_output_buffer)
 {
     d_logger->info(
-        "set_min_output_buffer on block {:s} to {:d}", unique_id(), min_output_buffer);
+        "set_min_output_buffer on block {:d} to {:d}", unique_id(), min_output_buffer);
```

Comment:
Most of the fmtlib examples don't use type specifiers. Wouldn't it be reasonable to just use `{}`?

Reply:
Most of the `boost::format` strings used explicit types, so I copied those over in the transition to fmtlib. C++20 is around the corner and will add type checking, so it's probably fine to leave the type specifiers in.